src/options.c

```diff
@@ -515,7 +517,7 @@ static void showUsage(void)
 {
     fputs(/* Check that everything lines up after any changes. */
         "usage:  " PACKAGE_NAME " [-bcfhimopuvz] [-a X,Y,W,H] [-C NAME] [-D DISPLAY]\n"
-        "              [-d SEC] [-e CMD] [-k OPT] [-l STYLE] [-M NUM] [-n OPTS]\n"
+        "              [-d SEC] [-e CMD] [-k OPT] [-l STYLE] [-M NUM]\n"
         "              [-q NUM] [-s OPTS] [-t % | WxH] [[-F] FILE]\n",
```

Comment:
`[-q NUM] [-s OPTS]` could be moved to the line above them, that would make it 79 characters long.

Reply:
In the output, yes. Those lines are already above 80 characters in the source code, actually.

The grey vertical bar is on column 81:
![2023-05-30-175737_828x189_scrot](https://github.com/resurrecting-open-source-projects/scrot/assets/25590950/8d47f9cc-7b1c-41c1-8f79-0ecfa43d69ca)

Reply:
Barely fits into 80chars now.

![image](https://github.com/resurrecting-open-source-projects/scrot/assets/79544946/81f71c89-24f2-4672-bf53-4bf373da74d0)

Also since the options now line up in the source itself, considering deleting that comment.